### DISCUSSION TITLE
Flutter and cockroachDB connection

### DISCUSSION THREAD

**Discussion by lokeshbs1994** [community/community#83873] [Mobile]
### Select Topic Area

Show & Tell

### Body

Please provide a sample project implementing the connectivity of flutter app with cockroachDB.

**Comment by esahmed21** [comment]
Hi @lokeshbs1994  👋 

Thank you for reaching out and sharing your thoughts. We appreciate your engagement with the community.

However, the focus of this community discussion is centered around our GitHub app. If you have any questions or concerns related to the app, please feel free to share them, and we'll be happy to assist.

For topics unrelated to our app, we recommend exploring other forums or communities that specialize in those areas.

Thanks for your understanding and cooperation!
